pki/src/test/java/org/hyperledger/besu/pki/keystore/SoftwareKeyStoreWrapperTest.java

```diff
@@ -171,7 +172,7 @@ public void getCertificateChainFallbackToTrustStore() throws Exception {
     when(trustStore.getCertificateChain(CERTIFICATE_ALIAS))
         .thenReturn(new Certificate[] {certificate});
 
-    assertThat(keyStoreWrapper.getCertificateChain(CERTIFICATE_ALIAS)).hasSize(1);
+    assertEquals(keyStoreWrapper.getCertificateChain(CERTIFICATE_ALIAS), 1);
```

Comment:
This should be using hasSize or similar so we check the array size and not compare Certificate[] with an int

Reply:
fixed